modules/nodes-vsphere-machine-set-scaling-static-ip.adoc

```diff
@@ -0,0 +1,158 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/node-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-vsphere-machine-set-scaling-static-ip_{context}"]
+= Using a machine set to scale machines with configured static IP addresses
```

Comment:
```suggestion
= Configuring machine set scaling of machines with static IP addresses
```

Reply:
I'll retain the original title as, IMO,  it's clearer